target_snowflake/__init__.py

```diff
@@ -231,7 +233,15 @@ def flush_records(stream, records_to_load, row_count, db_sync):
             f.write(bytes(csv_line + '\n', 'UTF-8'))
 
     s3_key = db_sync.put_to_stage(csv_file, stream, row_count)
-    db_sync.load_csv(s3_key, row_count)
+    try:
+        db_sync.load_csv(s3_key, row_count)
+    except Exception as e:
+        logger.error("""
+            Cannot load data from S3 into Snowflake schema: {} .
+            Try to delete {}.COLUMNS table to reset information_schema cache. Maybe it's outdated.
```

Comment:
Same, instead of asking to refresh cache manually, why not do it automatically? How often are we expecting to see that error? Pretty rare, right? But still, it would cause us to go to logs, find the error, and then act on the error, which is extra manual maintenance

Reply:
Same question as above. What should be the return code, success of failed?

Reply:
discussed this internally and we don't expect the code to fail at this point. We want to raise exception and exit with error if this happens.